src/Widgets/Canvas.vala

```diff
@@ -33,6 +33,7 @@ public class GtkCanvas.Canvas : Gtk.AspectFrame {
      */
     public signal void item_selected (CanvasItem item, Clutter.ModifierType modifiers);
 
+    public CanvasItem item;
```

Comment:
Don't declare item here :) You can declare the item inside of a function without initializing by just doing `CanvasItem item`